firebase-android-release-dashboard/src/components/ViewScheduledReleasesButton/ViewScheduledReleasesButton.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import {Button, Grid, Typography} from "@material-ui/core";
+import {Schedule} from "@mui/icons-material";
+import PropTypes from "prop-types";
+import useStyles from "./styles";
+
+/**
+ * ViewScheduledReleasesButton is a button that opens the ScheduledReleasesModal
+ *
+ * @param {Function} props.onClick - The function to call when the button is
+ * clicked.
+ * @return {JSX.Element} - The ViewScheduledReleasesButton component.
+ */
+function ViewScheduledReleasesButton({onClick}) {
+  const classes = useStyles();
+
+  return (
+    <Button
```

Comment:
A bit of extra bottom padding on this button would be good, also, I'm noticing the icon isn't centered with the text, and neither is centered within the button

Reply:
Fixed in https://github.com/firebase/firebase-release-dashboard/pull/14/commits/0a7b953c0110751134ee042d351d0678317f921f